app/main.ipynb

```diff
@@ -275,17 +431,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "outputs": [],
+   "execution_count": 58,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "                Unnamed: 0    age  hours-per-week  salary K$\nUnnamed: 0           False  False           False      False\nage                  False  False           False      False\nhours-per-week       False  False           False      False\nsalary K$            False  False           False      False",
+      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>Unnamed: 0</th>\n      <th>age</th>\n      <th>hours-per-week</th>\n      <th>salary K$</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>Unnamed: 0</th>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n    </tr>\n    <tr>\n      <th>age</th>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n    </tr>\n    <tr>\n      <th>hours-per-week</th>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n    </tr>\n    <tr>\n      <th>salary K$</th>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n      <td>False</td>\n    </tr>\n  </tbody>\n</table>\n</div>"
+     },
+     "execution_count": 58,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code for task 9 here"
+    "num_data = clean_data._get_numeric_data()\n",
+    "\n",
+    "corr_data = num_data.corr()\n",
+    "\n",
+    "s_corr = (corr_data.abs() > 0.5) & (corr_data.abs() < 1)\n",
+    "s_corr "
```

Comment:
you should show all correlation data even if correlation is <= 0.5 or correlation == 1(it can't be bigger than 1 anyway) in order to know the connection between different data columns